test/sequential/sequencerStaking.cancel.test.ts

```diff
@@ -1,403 +1,281 @@
-    expect(slashRewardUpdater).bnEqual(BN_ZERO);
-    expect(slashFineCanceler).bnEqual(await SequencerStaking.slashFineAmount());
-  });
+  expect(updaterDiffValue).bnEqual(BN_ZERO);
+  expect(cancelerPenaltyValue).bnEqual(
+    await SequencerStaking.slashFineAmount(),
+  );
+});
 
-  it("GIVEN a sequencer, WHEN <no> canceling an update THEN no slash is applied", async () => {
-    await testUser2.refreshAmounts(AssetWallet.BEFORE);
-    await testUser1.refreshAmounts(AssetWallet.BEFORE);
-    await createAnUpdateAndCancelIt(testUser1, testUser2, chain, false);
-    await waitForNBlocks((await Rolldown.disputePeriodLength()).toNumber());
-    await testUser2.refreshAmounts(AssetWallet.AFTER);
-    await testUser1.refreshAmounts(AssetWallet.AFTER);
-    const slashRewardUpdater = testUser1
-      .getAsset(MGA_ASSET_ID)
-      ?.amountAfter.free!.sub(
-        testUser1.getAsset(MGA_ASSET_ID)?.amountBefore.free!,
-      );
-    const slashFineUpdater = testUser1
-      .getAsset(MGA_ASSET_ID)
-      ?.amountBefore.reserved!.sub(
-        testUser1.getAsset(MGA_ASSET_ID)?.amountAfter.reserved!,
-      );
-    expect(slashRewardUpdater).bnEqual(BN_ZERO);
-    expect(slashFineUpdater).bnEqual(BN_ZERO);
-  });
+it("GIVEN a sequencer, WHEN <no> canceling an update THEN no slash is applied", async () => {
+  await testUser1.refreshAmounts(AssetWallet.BEFORE);
+  await createAnUpdate(testUser1, chain);
+  await waitForNBlocks(disputePeriodLength);
+  await testUser1.refreshAmounts(AssetWallet.AFTER);
+  const updaterPenaltyValue = testUser1
+    .getAsset(GASP_ASSET_ID)
+    ?.amountBefore.reserved!.sub(
+      testUser1.getAsset(GASP_ASSET_ID)?.amountAfter.reserved!,
+    );
+  expect(updaterPenaltyValue).bnEqual(BN_ZERO);
+});
+
+it("GIVEN a slashed sequencer, WHEN slashed it can not provide any update / cancel until the next session ( if gets elected )", async () => {
+  let updaterRightsStatus: any;
+  const { reqIdCanceled, api } = await createAnUpdateAndCancelIt(
+    testUser1,
+    testUser2Address,
+    chain,
+  );
+  await waitForNBlocks(disputePeriodLength);
+  updaterRightsStatus = await Rolldown.sequencerRights(
+    chain,
+    testUser1.keyRingPair.address,
+  );
+  expect(updaterRightsStatus.cancelRights.toString()).toBe("1");
+  const txIndex = await Rolldown.lastProcessedRequestOnL2(chain);
+  //we approve the cancellation
+  await Rolldown.waitForReadRights(testUser2Address);
+  const cancelResolutionEvents = await Sudo.asSudoFinalized(
+    Sudo.sudoAsWithAddressString(
+      testUser2Address,
+      new L2Update(api)
+        .withCancelResolution(txIndex, reqIdCanceled, true)
+        .on("Ethereum")
+        .build(),
+    ),
+  );
+  await waitSudoOperationSuccess(cancelResolutionEvents, "SudoAsDone");
+  await waitForNBlocks(disputePeriodLength);
+  updaterRightsStatus = await Rolldown.sequencerRights(
+    chain,
+    testUser1.keyRingPair.address,
+  );
+  expect(updaterRightsStatus.cancelRights.toString()).toBe("0");
```

Comment:
plz additionally validate that ,  
- it does no longer belong to the active set,
- it does not have read rights anymore

Reply:
Done

Reply:
ty!